export/export.go

```diff
@@ -63,3 +66,20 @@ func ExportBranchDone(cataloger catalog.Cataloger, status catalog.CatalogBranchE
 	})
 	return err
 }
+
+var ErrRepairNonFailed = errors.New("branch status is not export-failed, could not be repaired")
```

Comment:
```suggestion
var ErrRepairWrongStatus = errors.New("incorrect status")
```
and then state the specifics on a wrapping error.  Otherwise we could have many more errors of very similar type -- and the client needs to handle them all in exactly the same way.

This might even return as a separate HTTP code.